app/services/spree_shopify_importer/connections/base.rb

```diff
@@ -8,7 +8,7 @@ def count(**opts)
 
         def all
           results = []
-          batch = api_class.find(:all, params: { limit: 2 })
+          batch = api_class.find(:all, params: { limit: 50 })
```

Comment:
are you sure that it's not big payload to get 50 records by 1 request?

Reply:
I'm not, but it is a default value, and the maximum is 250, 2 was just because I had only 3 orders when I did it, and I need 2 as the limit to verification

Reply:
got it, thanks